AirCasting/MicrophoneSession/MicrophoneManager.swift

```diff
@@ -122,9 +143,15 @@ class MicrophoneManager: ObservableObject {
             session.addToMeasurementStreams(measurementStream!)
         }
     }
-
+    
     func obtainCurrentLocation() -> CLLocationCoordinate2D {
         return locationProvider.currentLocation?.coordinate ?? CLLocationCoordinate2D(latitude: 200.0, longitude: 200.0)
     }
     
+    func addInterruptionsObserver() {
```

Comment:
there is no removal of the notification. if the microton manage would be only one or would exist only for the time being of the record it maybe enough. But if there would ever be a different object that is using `AVAudioSession` then  its interruption would influence this object. also maybe pass in the `audioSession` as the `object:` argument and not nil then the Notification Center will not call this observer when the object is not the seam as the one calling it.

Reply:
Where should the removal be implemented?